### DISCUSSION TITLE
Error running Whisper

### DISCUSSION THREAD

**Discussion by ghost** [openai/whisper#1444] [Q&A]
Hello,
I tried to run Whisper on my Windows PC but I get this error, do you know how to fix the problem?
Thanks

`PS E:\whisper> whisper "Test.mp4"
whisper : C:\Python311\Lib\site-packages\whisper\timing.py:57: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See 
https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
Au caractère Ligne:1 : 1
+ whisper "Test.mp4"
+ ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (C:\Python311\Li...or details.[0m:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
  @numba.jit
C:\Python311\Lib\site-packages\whisper\transcribe.py:114: UserWarning: FP16 is not supported on CPU; using FP32 instead
  warnings.warn("FP16 is not supported on CPU; using FP32 instead")`

**Comment by phineas-pta** [comment]
this is the error when running python with powershell ise

run with powershell (not ise) or cmd
